packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -342,6 +342,9 @@ Resources:
               - ecr:BatchCheckLayerAvailability
               - ecr:DescribeRepositories
               - ecr:DescribeImages
+              - ecr:BatchGetImage
+              - ecr:GetDownloadUrlForLayer
+              - ecr:InitiateLayerUpload
```

Comment:
`ecr:InitiateLayerUpload` is duplicated here.

Do you know why this wasn't detected earlier? Seems like a big miss if true...

Reply:
I really don't know... I only recently switched all my productions stack to the new style stack synthesis and came upon this.

While perfectly valid and working, I'm not sure that the pattern is commonly used...

Reply:
I guess the issue only crops up when using an ECR image as a base, huh?

Reply:
Yes